### PR TITLE
Admin redirect#90

### LINKED ISSUE #90: User Story 44, Admin is redirected from Merchant Dashboard to User profile
```
As an admin user
If I visit a merchant dashboard, but that merchant is a regular user
Then I am redirected to the appropriate user profile page.

eg, if I visit "/admin/merchants/7" but that merchant is a regular user
then I am redirected to "/admin/users/7" and see their user profile page
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #97

### PR DESCRIPTION
**Pull request by stoic-plus**
closes #90 and #112 

* Admin is redirected to `admin_user` if passed id corresponds to a default user
* Admin is redirected to `admin_merchant` if passed id corresponds to a merchant

### CODE REVIEW

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ def index
 
   def show
     @user = User.find(params[:id])
+    redirect_to admin_user_path(params[:id]) if @user.default?
```
Comment (tonyr729) [inline app/controllers/admin/merchants_controller.rb]: Weird feature
